features/FxA-105-ip-blocklist/README.md

```diff
@@ -191,19 +191,32 @@ Additionally, the module will be able to load and refresh multiple lists.
 
 ### Phase 1
 
-When Phase 1 is complete,
-we will include here
-a snapshot of the hit-rate graph
-for the initial set of blocklists.
+Phase 1 was completed and it analyzed two blocklists, [FireHOL Level 1](http://iplists.firehol.org/?ipset=firehol_level1#)
+and [FireHOL webserver](http://iplists.firehol.org/?ipset=firehol_webserver#). The graph below shows potential hits for
+two days worth of real FxA network traffic.
+
+<img src="blocklist_hits.png" />
+
+While it appears that both lists would block zero requests, it is really
+a non-zero number but still very small compared to all network traffic.
+
+Calulated hit rate %:
+
+* FireHOL Level 1 - 0.001% Hit rate
+* FireHOL Webserver - 0.55% Hit rate
+
+These results make sense because FireHOL Level 1 list is the most
+conservative blocklist. Addresses on this list are usually deemed
+malicious by multiple sources.
 
-We will also calculate,
-but not share publicly,
-the absolute number of hits per day
-on each blocklist.
+FireHOL Webserver list is more liberal and therefore had a higher hit rate.
+We decided to run some analysis on the ip addresses that would have been
+blocked from this list and the majority of these were coming from ip
+addresses usually reserved for AWS EC2 instances. There is a possiblity
+that some of these requests were malicious because we wouldn't
+normally expect network traffic from an EC2 instance.
```

Comment:
IIUC, part of the issue here is that we *do* normally expect some amount of traffic from EC2 instances, e.g. for automated testing infra.  So we didn't enable this list because we didn't want to break tests.